src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -6941,9 +6941,7 @@ internal static List<CompletionResult> CompleteStatementFlags(TokenKind kind, st
                     bool withColon = wordToComplete.EndsWith(':');
                     wordToComplete = withColon ? wordToComplete.Remove(wordToComplete.Length - 1) : wordToComplete;
 
-                    string enumString = LanguagePrimitives.EnumSingleTypeConverter.EnumValues(typeof(SwitchFlags));
-                    string separator = CultureInfo.CurrentUICulture.TextInfo.ListSeparator;
-                    string[] enumArray = enumString.Split(separator, StringSplitOptions.RemoveEmptyEntries);
+                    string[] enumArray = LanguagePrimitives.EnumSingleTypeConverter.GetEnumNames(typeof(SwitchFlags));
```

Comment:
There is one more place where we can use the helper if I remember correctly.

Reply:
This is the one you mentioned, not covered by #17750 . I don't see other references besides log/exception-messages where they want this comma-separated string.

Reply:
Please fix both in the PR.

Reply:
? No need to update code I've suggested to remove in #17750

Even the line changed might be be unnecessary due to #17684